api/include/opentelemetry/common/key_value_iterable.h

```diff
@@ -19,16 +19,31 @@ class KeyValueIterable
    * Iterate over key-value pairs
    * @param callback a callback to invoke for each key-value. If the callback returns false,
    * the iteration is aborted.
-   * @return true if every key-value pair was iterated over
+   * @return true if every key-value pair was iterated ove
    */
-  virtual bool ForEachKeyValue(
-      nostd::function_ref<bool(nostd::string_view, common::AttributeValue)> callback) const
-      noexcept = 0;
+  virtual bool ForEachKeyValue(nostd::function_ref<bool(nostd::string_view, common::AttributeValue)>
+                                   callback) const noexcept = 0;
 
   /**
    * @return the number of key-value pairs
    */
   virtual size_t size() const noexcept = 0;
 };
+
```

Comment:
There should be no need to add that. I'll add in a separate commit. Not needed for ETW exporter.

Reply:
removed